starter-code/pizza.js

```diff
@@ -1 +1,44 @@
 // Write your Pizza Builder JavaScript in this file.
+let $pepperonis = $('.pep');
+let $mushrooms = $('.mushroom');
+let $greenPeppers = $('.green-pepper');
+let $crust = $(`.crust`);
+let $sauce = $(`.sauce`);
+
+let $pepperonisBtn = $('.btn-pepperonni');
+let $mushroomsBtn = $('.btn-mushrooms');
+let $greenPeppersBtn = $('.btn-green-peppers');
+let $crustBtn = $(".btn-crust");
+let $sauceBtn = $(".btn-sauce");
+
```

Comment:
Can add the toggle function around here for all ingredients you don't want displayed when the page loads.